assets/README.md

```diff
@@ -60,3 +60,62 @@ run:
 ```bash
 npm run test:watch
 ```
+
+## Run a Local Version on a Mobile Device
+Disconnect from Virtual Private Network (VPN) or USGS network.
+
+Install a 'tunneling' application (LocalTunnel is an example (this is not an endorsement)).
+
+Start the tunneling application; make a tunnel for each server
+
+```bash 
+An example using LocalTunnel 
+(each command is in a seprate terminal, since each will be a running process)
+lt --port 5050 (for the Flask server)
+lt --port 9000 (for the Static Assets server)
+You will get back two URLs looking something like -- https://bad-lionfish-7.loca.lt
+```
+Take the URL returned for the Static Assets server and go to the 'instance/config,py'.
+
+You will only have an 'instance/config.py' if you created one using
+```bash 
+(as described in the WDFN-server setup)
+mkdir -p instance
+cp config.py.sample instance/config.py
+```
+If you don't have an 'instance/config' make one and then replace the value of 'STATIC_ROOT' with
+the URL returned for the Static Assets server. 
+
+Once you have changed the STATIC_ROOT, go to 'wdfn-server/config.py' and find the code
+```bash
+if os.getenv('CONTAINER_RUN', False):
+```
+Change 'False'  to 'True' (noting the capitalization).
```

Comment:
I would not change the wdfn-server/config.py. This risks committing the config file with this change. Rather set an environment variable, CONTAINER_RUN to True.